niworkflows/interfaces/tests/test_images.py

```diff
@@ -3,8 +3,52 @@
 import nibabel as nb
 from nipype.interfaces import nilearn as nl
 from .. import images as im
+from pathlib import Path
 
 import pytest
+qform_code = 1
+sform_code = 1
+qform = np.eye(4)
+                  [0, 1,  0, 0],
+                  [0,  0,  1, 0],
+                  [0.00000000e+00,   0.00000000e+00,   0.00000000e+00, 1.00000000e+00]])
```

Comment:
@utooley You'll need to drop the other three lines from each of these. L12-14 and L16-18.